arch.mmd

```diff
@@ -0,0 +1,23 @@
+%% Ensure when adding entities here they match what's defined in the internal-docs repo
+
+subgraph Core
+  core --> core_maria_db
+end
+
+subgraph Resolver
+  resolver --> resolver_maria_db
+end
+
+subgraph Device Registry
+  device_registry --> device_registry_maria_db
+end
+
+core --> auth_plus
+core --> device_registry
+core --> kafka
+core --> package_store
+core --> resolver
+core --> tuf_repo
+core --> user_profile
+device_registry --> kafka
+resolver --> core
```

Comment:
So if we don't distinguish between listens vs. publish then I suppose `resolver` should also point to `kafka` since it does listen.

Yes resolver talks to device_registry but not core.